python/paddle/tensor/__init__.py

```diff
@@ -11,205 +11,315 @@
-from .math import mm  #DEFINE_ALIAS
-from .math import divide  #DEFINE_ALIAS
-from .math import floor_divide  #DEFINE_ALIAS
-from .math import remainder  #DEFINE_ALIAS
-from .math import mod  #DEFINE_ALIAS
-from .math import floor_mod  #DEFINE_ALIAS
-from .math import multiply  #DEFINE_ALIAS
-from .math import add  #DEFINE_ALIAS
-from .math import subtract  #DEFINE_ALIAS
-from .math import atan  #DEFINE_ALIAS
-from .math import logsumexp  #DEFINE_ALIAS
-from .math import inverse  #DEFINE_ALIAS
-from .math import log2  #DEFINE_ALIAS
-from .math import log10  #DEFINE_ALIAS
-from .math import log1p  #DEFINE_ALIAS
-from .math import erf  #DEFINE_ALIAS
-from .math import addmm  #DEFINE_ALIAS
-from .math import clip  #DEFINE_ALIAS
-from .math import trace  #DEFINE_ALIAS
-from .math import kron  #DEFINE_ALIAS
-from .math import isfinite  #DEFINE_ALIAS
-from .math import isinf  #DEFINE_ALIAS
-from .math import isnan  #DEFINE_ALIAS
-from .math import prod  #DEFINE_ALIAS
-from .math import all  #DEFINE_ALIAS
-from .math import any  #DEFINE_ALIAS
-from .math import broadcast_shape  #DEFINE_ALIAS
-from .math import conj  #DEFINE_ALIAS
+from .array import array_length  # noqa: F401
+from .array import array_read  # noqa: F401
+from .array import array_write  # noqa: F401
+from .array import create_array  # noqa: F401
 
-from .random import multinomial  #DEFINE_ALIAS
-from .random import standard_normal
-from .random import normal
-from .random import uniform  #DEFINE_ALIAS
-from .random import randn  #DEFINE_ALIAS
-from .random import rand  #DEFINE_ALIAS
-from .random import randint  #DEFINE_ALIAS
-from .random import randperm  #DEFINE_ALIAS
-from .search import argmax  #DEFINE_ALIAS
-from .search import argmin  #DEFINE_ALIAS
-from .search import argsort  #DEFINE_ALIAS
-# from .search import has_inf  #DEFINE_ALIAS
-# from .search import has_nan  #DEFINE_ALIAS
-# from .search import masked_select        #DEFINE_ALIAS
-from .search import topk  #DEFINE_ALIAS
-from .search import where  #DEFINE_ALIAS
-from .search import index_select  #DEFINE_ALIAS
-from .search import nonzero  #DEFINE_ALIAS
-from .search import sort  #DEFINE_ALIAS
-from .search import index_sample  #DEFINE_ALIAS
-from .search import masked_select  #DEFINE_ALIAS
-from .stat import mean  #DEFINE_ALIAS
-# from .stat import reduce_mean  #DEFINE_ALIAS
-from .stat import std  #DEFINE_ALIAS
-from .stat import var  #DEFINE_ALIAS
-from .stat import numel  #DEFINE_ALIAS
-from .stat import median  #DEFINE_ALIAS
-# from .tensor import Tensor        #DEFINE_ALIAS
-# from .tensor import LoDTensor        #DEFINE_ALIAS
-# from .tensor import LoDTensorArray        #DEFINE_ALIAS
-from .to_string import set_printoptions  #DEFINE_ALIAS
-
-from .array import array_length  #DEFINE_ALIAS
-from .array import array_read  #DEFINE_ALIAS
-from .array import array_write  #DEFINE_ALIAS
-from .array import create_array  #DEFINE_ALIAS
+#this list used in math_op_patch.py for _binary_creator_
```

Comment:
They are not used by `_binary_creator_` to create methods for `paddle.Tensor`. They are imported as-is and attached to `paddle.Tensor` as methods.

Also, many operations listed is not binary (like abs, acos, ...). I suggest changing the name.

Reply:
done, binary_ops->tensor_method_func